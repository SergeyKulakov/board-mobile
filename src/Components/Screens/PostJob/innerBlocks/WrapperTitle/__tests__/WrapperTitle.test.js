import React from 'react'
import { shallow } from 'enzyme'

import 'jest-styled-components'

import { WrapperTitle as Component } from '../index'

describe('PostJob.WrapperTitle', () => {
  const defaultProps = {
    isError: false,
  }

  it('render snapshot', () => {
    const wrapper = shallow(<Component {...defaultProps}>title</Component>)

    expect(wrapper).toMatchSnapshot()
  })
})
