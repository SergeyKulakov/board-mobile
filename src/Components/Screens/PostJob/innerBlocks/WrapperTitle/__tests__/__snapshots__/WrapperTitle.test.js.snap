// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostJob.WrapperTitle render snapshot 1`] = `
<Styled(Text)
  error={false}
>
  title
</Styled(Text)>
`;
